main.py

```diff
@@ -0,0 +1,3 @@
+def return_4():
```

Comment:
Make sure to add a docstring to the file and a docstring to the function even if it is for testing. The function docstring can mention how this is a function for testing

Reply:
I did this to get a specific pylint score, so I could test the pylint acts as expected.